src/app/clusters/scenes-server/SceneTableImpl.cpp

```diff
@@ -738,7 +738,8 @@ CHIP_ERROR DefaultSceneTableImpl::SceneSaveEFS(SceneTableEntry & scene)
     if (!HandlerListEmpty())
     {
         uint8_t clusterCount = 0;
-        clusterId cArray[kMaxClustersPerScene];
+        // TODO: Use dynamical allocation with emberAFClusterCount to generate a smaller array
+        clusterId cArray[GENERATED_CLUSTER_COUNT];
```

Comment:
So we are not supporting dynamic endpoint. Is there prevention/protection agasint dynamically created endpoints from accessing this array?